rohit/widgets/src/components/Dropdown.js

```diff
@@ -51,7 +51,7 @@ const Dropdown = ({ options, selected, onSelectedChange }) => {
         </div>
       </div>
     </div>
-  );  
+  );
```

Comment:
Thank you for removing this trailing space. It's important to keep the code clean.